versioned_docs/version-6.x.x/core-concepts/validations-and-constraints.md

```diff
@@ -158,19 +155,17 @@ isIn: {
 }
 ```
 
-When using custom validator functions the error message will be whatever message the thrown `Error` object holds.
-
-See [the validator.js project](https://github.com/chriso/validator.js) for more details on the built in validation methods.
+When using custom validator functions, the error message will be whatever message the thrown `Error` object holds.
 
 **Hint:** You can also define a custom function for the logging part. Just pass a function. The first parameter will be the string that is logged.
 
-### `allowNull` interaction with other validators
+### `allowNull: true` with custom validators
 
-If a particular field of a model is set to not allow null (with `allowNull: false`) and that value has been set to `null`, all validators will be skipped and a `ValidationError` will be thrown.
+If a field that does not allow null values is set to null, all validators will be skipped. A `ValidationError` will be thrown.
 
-On the other hand, if it is set to allow null (with `allowNull: true`) and that value has been set to `null`, only the built-in validators will be skipped, while the custom validators will still run.
+If a field that allows null values (`allowNull: true`) is set to null, only the built-in validators will be skipped. Custom validators will still run.
 
-This means you can, for instance, have a string field which validates its length to be between 5 and 10 characters, but which also allows `null` (since the length validator will be skipped automatically when the value is `null`):
+This can be useful, for example, if you have a string field that can have a length between 5 and 10 characters but also allows `null`. If the value is `null`, the length validator is skipped:

```

Comment:
```suggestion
This may be useful for string fields that allow 5 to 10 characters but also `null`. If the value is `null`, the length validator is skipped:
```